docs/awsome.md

```diff
@@ -0,0 +1,154 @@
+
+Please take a look at the existing charts before naming your own.
+
+_A Note on example charts:_ As Helm was getting started, we created a few example charts, whose purpose was to illustrate how to write a chart. We are trying to only add new _example_ charts when they illustrate something new and helpful for chart developers.
+
+## The Chart.yaml File
+
+A simple Chart.yaml file should look like this:
+
+```
+name: NAME OF CHART
+home: URL TO IMAGE HOME
+version: VERSION
+description: SHORT DESCRIPTION
+dependencies:
+  - name: CHART
+    version: SEMVER FILTER
+maintainers:
+  - YOUR NAME <YOUR EMAIL>
+details:
+  ONE PARAGRAPH DESCRIPTION
+```
+
+- name: This should be the same name as your chart directory.
+- home: The URL to the project where the container image came from. This is to assist people in finding out how their image was built.
+- version: A SemVer 2 version strings.
+- description: A short (several word) description of this chart.
+- dependencies: A set of name/filter pairs
+	- name: The name of the chart this chart depends upon
+	- version: A filter indicating what version of the chart is required. Example: `~1.2` (greater than or equal to 1.2.0, and less than 1.3.0)
+- maintainers: A set of maintainer names, together with an email
+- details: A single paragraph describing the chart
+
+Except for `dependencies`, all fields are required.
+
+## The README.md File
+
+The README file performs one important function: It tells the user how to use your chart. It is automatically displayed when a chart is installed.
+
+Your README.md file _may_:
+
+- Tell the user how to configure the chart (if necessary)
+- How to work with the chart once it's in Kubernetes (if this is not self-evident)
+- Explain any environment variables, secrets, annotations, or special labels
+- Provide URLs to external sources that are helpful
+
+Your REDME.md file _should not_:
+
+- Add a license constraint (all charts in `deis/charts` are Apache 2)
+- Describe the project in detail
+
+In a nutshell, the README is intended as help text for your user: It gets the user from fetching your chart to using your chart.
+
+## Manifest Files
+
+All Kubernetes manifest files should be in YAML format. We know some people prefer JSON, but we've decided for a number of reasons to adopt YAML as the single format for Helm manifests.
+
+A good simple manifest typically includes:
+
+- A replication controller (`foo-rc.yaml`)
+- A service definition for each backend service this chart relies upon (`foo-db-svc.yaml`)
+- A secrets file for each secret the RC uses (`foo-password-sec.yaml`)
+
+### Labels
+
+All Helm charts should have a `name` label and a `heritage: helm` label in their metadata sections. These provide a base-level consistency across all Helm charts. (`heritage: helm` makes it easy to search a Kubernetes cluster for all components installed via Helm.)
```

Comment:
The Kubernetes team has been leaning in favor of using the `app` label instead of `name`, probably due to confusion between a "name" label and the "name" metadata attribute.